bcache/get.go

```diff
@@ -0,0 +1,329 @@
+// Copyright 2017 The Prometheus Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package bcache
+
+import (
+	"bufio"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path"
+	"path/filepath"
+	"strconv"
+	"strings"
+)
+
+// parsePseudoFloat parses the peculiar format produced by bcache's bch_hprint
+func parsePseudoFloat(str string) (float64, error) {
+	ss := strings.Split(str, ".")
+
+	intPart, err := strconv.ParseFloat(ss[0], 64)
+	if err != nil {
+		return 0, err
+	}
+
+	if len(ss) == 1 {
+		// Pure integers are fine
+		return intPart, nil
+	}
+	fracPart, err := strconv.ParseFloat(ss[1], 64)
+	if err != nil {
+		return 0, err
+	}
+	// fracPart is a number between 0 and 1023 divided by 100; it is off
+	// by a small amount. Unexpected bumps in time lines may occur because
+	// for bch_hprint .1 != .10 and .10 > .9 (at least up to Linux
+	// v4.12-rc3).
+
+	// Restore the proper order:
+	fracPart = fracPart / 10.24
+	return intPart + fracPart, nil
+}
+
+// dehumanize converts human-readable byte slice into float64
```

Comment:
Also, we have a comment style for complete sentences.  Everything should start caps, and end with a period.